packages/component-library/src/MapTooltip/MapTooltip.js

```diff
@@ -24,6 +24,7 @@ const MapTooltip = (props) => {
     primaryField,
     secondaryName,
     secondaryField,
+    isHex
```

Comment:
Ideally we don't have component-specific switches like this in general components. If the hex overlay needs its own code path, that is good evidence that the abstractions aren't right.

Either hex overlay needs its own tooltip or maptooltip's api needs to expand in a way to support more use cases.

@dorenproctor and I had a conversation about this on Monday.

Reply:
Currently MapTooltip does require isHex to be true like you pointed out:

    <MapTooltip isHex={true} />

We can discuss alternatives tomorrow and/or we can work on new StoryCards like we discussed and consider this later. Either way is fine with me, let's discuss tomorrow. Thanks.